src/geometry/manifold/manifold-applyops.cc

```diff
@@ -19,6 +19,16 @@ Location getLocation(const std::shared_ptr<const AbstractNode>& node)
   return node && node->modinst ? node->modinst->location() : Location::NONE;
 }
 
+std::unique_ptr<Geometry> applyHullManifold(const Geometry::Geometries& children)
+{
+  std::vector<manifold::Manifold> manifolds;
+  for (const auto& item : children) {
+    auto chN = item.second ? createManifoldFromGeometry(item.second) : nullptr;
+    if (chN) manifolds.emplace_back(chN->getManifold());
+  }
+  return std::make_unique<ManifoldGeometry>(manifold::Manifold::Hull(manifolds));
```

Comment:
One of the benefits of using hull() is that the underlying geometry doesn't have to be manifold (e.g. for imported or manually constructed objects). Manifold offers a public function taking only vertices as argument (you're using it in the Minkowski code path). Did you (or should we) consider using that instead and rather extract vertices instead of constructing a Manifold geometry first?

The test case difference also illustrates this point :)